tendrl/ceph_integration/objects/ecprofile/atoms/check_ec_profile_available/__init__.py

```diff
@@ -0,0 +1,45 @@
+import etcd
+import gevent
+
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons import objects
+
+
+class CheckECProfileAvailable(objects.BaseAtom):
+    def __init__(self, *args, **kwargs):
+        super(CheckECProfileAvailable, self).__init__(*args, **kwargs)
+
+    def run(self):
+        retry_count = 0
+        while True:
+            try:
+                rbd = NS._int.client.read(
```

Comment:
use object.load() , we need to ensure all attributes inside the object are also present as provided by user